bodyshopbidsdotcom/git-scripts
# Remove not required args

### How to reproduce error?
1. Checkout `release` branch of `git-scripts`
2. Checkout some project repo like `hermes` and run `git_terraform_commands production --production --network`
3. You will see error `network_command() takes exactly 3 arguments (1 given)`

### Now how to test the fix?
1. Checkout `this` branch of `git-scripts`
2. Checkout some project repo like `hermes` and run `git_terraform_commands production --production --network`
3. You won't see any error
